confgenerator/testdata/invalid/linux/metrics-receiver_postgresql_insecure_plus/golden_error

```diff
@@ -0,0 +1 @@
+when insecure is true, no other TLS fields may be configured. Values were found for field(s) insecure_skip_verify, cert_file, ca_file, key_file for receiver postgresqlmetrics
```

Comment:
Slight adjustment for error message convention compliance:

“insecure_skip_verify“, ”cert_file“, ”ca_file“, ”key_file“ are not allowed when "insecure" is true, which indicates TLS is disabled for receiver "postgresqlmetrics".

Reply:
Updated.